server/views/pages/search/caSearch/caSearch.njk

```diff
@@ -96,9 +96,49 @@
         text: "Release date",
         attributes: {
             id: 'release-date-sort',
+            "aria-sort": "ascending"
+        }
+    },
+    {
+        text: "Last worked on by",
+        attributes: {
+            "aria-sort": "none"
+        }
+    },
+    {
+        text: "Status",
+        attributes: {
+            "aria-sort": "none"
+        }
+    }
+] %}
+
+{% set onProbationSearchTableHeadings = [
+    {
+        text: "Name",
+        attributes: {
             "aria-sort": "none"
         }
     },
+    {
+        text: "Prison number",
+        attributes: {
+            "aria-sort": "none"
+        }
+    },
+    {
+        text: "Probation practitioner",
+        attributes: {
+            "aria-sort": "none"
+        }
+    },
+    {
+        text: "Release date",
+        attributes: {
+            id: 'release-date-sort',
+            "aria-sort": "descending"
+        }
+    },
```

Comment:
Implemented two searchable table headings for each tab as in prison results are sorted by `"aria-sort": "descending"` and on probation are sorted by `"aria-sort": "ascending"`. The area-sort changes here indicate sorting to the user by showing 🔺 or 🔻 . Tests to check sorting results are pre existing.

Reply:
Just edited the integration test 👍